osquery/core/system.cpp

```diff
@@ -94,20 +94,21 @@ struct tm* localtime_r(time_t* t, struct tm* result) {
   _localtime64_s(result, t);
   return result;
 }
+
 #endif
 
 std::string getHostname() {
 #ifdef WIN32
-  long size = 256;
+  long size = MAX_PATH;
```

Comment:
Er not sure we can, as `MAX_PATH` isn't defined on posix systems. I can manually define something for `MAX_PATH`, but that might cause conflicts with Windows builds.